validation/src/main/resources/esapi/ESAPI.properties

```diff
@@ -466,9 +466,13 @@ Validator.HTTPURL=^.*$
 Validator.HTTPJSESSIONID=^[A-Z0-9]{10,32}$
 
 Validator.SITEID=^[a-z0-9\-_]*$
+Validator.EMAIL=^([\\w\\d._\\-#])+@([\\w\\d._\\-#]+[.][\\w\\d._\\-#]+)+$
 Validator.USERNAME=^[a-zA-Z0-9_\\.]{1,255}$
 Validator.ALPHANUMERIC=^[a-zA-Z0-9]*$
 
+# Validation of SQL related input
+Validator.SQL_ORDER_BY=^([a-zA-Z_][a-zA-Z_0-9]*,?)+(\\sasc|\\sdesc)?$
```

Comment:
I believe these sort order (asc|desc part) should be case-insensitive as well
cc: @sumerjabri